src/test/java/com/wikia/webdriver/testcases/desktop/discussions/UploadingImageTests.java

```diff
@@ -20,19 +19,20 @@
 import java.net.MalformedURLException;
 import java.net.URL;
 
+@Execute(onWikia = "qadiscussions", language = "de")
 public class UploadingImageTests extends NewTestTemplate {
 
+  private static final String wikiName = "qadiscussions";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)